### ISSUE TITLE
Merging overlapping tooltip position is off

### ISSUE THREAD

**Issue by bjarnef** [leongersen/noUiSlider#1217]
When merging overlapping tooltips was introduced it looked great https://github.com/leongersen/noUiSlider/issues/1032
https://refreshless.com/nouislider/examples/#section-merging-tooltips

But it seems something has changed in recent updates as the position is off.

The example here is based on **v15.6.0**

https://user-images.githubusercontent.com/2919859/182472435-8e70971f-3279-4496-bef6-f31ffbca3501.mp4

When I revert back to **v15.4.0** it starts working again.

https://user-images.githubusercontent.com/2919859/182475079-ae607ad3-cb5d-45ec-b10c-cc951a5e97af.mp4

Not sure which change broke this feature though, but it seems to be related to one on the changes v15.5.0 where I could reproduce the issue as well.

**Comment by leongersen** [comment]
This is an unfortunate side effect of a compatibility fix in [15.5.0](https://github.com/leongersen/noUiSlider/blob/master/CHANGELOG.MD#1550-2021-09-01).

**TLDR**: [these changes in the default stylesheet](https://github.com/leongersen/noUiSlider/commit/a1f8f17fa7a2a2ea81a8f9a4fd7c4692ebe16927) need to be incorporated in your overriding styles.

See #1167 for more discussion on this.

**Comment by bjarnef** [comment]
Yes, we include the `nouislider.min.css` and overwrite a bit of the styling.
But not regarding `height` and `width` on `.@{noUi-css-prefix}-connect` and `.@{noUi-css-prefix}-origin`

https://github.com/leongersen/noUiSlider/commit/a1f8f17fa7a2a2ea81a8f9a4fd7c4692ebe16927

When I remove the custom stylesheet overwriting the default CSS I still get this:

https://user-images.githubusercontent.com/2919859/182635980-4ca0c457-6ee9-45bb-beed-4e03b61546ca.mp4

What am I missing?

**Comment by leongersen** [comment]
The example on merging overlapping tooltips [was also updated](https://github.com/leongersen/noUiSlider/commit/607c4728bf660d12611490fd44b2f765c67947bb#diff-7fad8517c52306464138093804724b7893383750e52c943fe6f35c3fc368e9fd) to accommodate for the sizing changes, you'll want to upgrade that, too. I'll make sure to flag that in the changelog.
